### ISSUE TITLE
Opening a popup window on android doesn't always focus first item

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#18538]
Possibly due to the special case of scopesRoute.

Edit: removing the special case of no focus allowed on a scopesRoute node fixes this.  Will investigate more

**Comment by jonahwilliams** [comment]
Possibly also on iOS

**Comment by jonahwilliams** [comment]
There also doesn't seem to be a way to dismiss the popup
